### DISCUSSION TITLE
Latency not set in “you” Battlesnake object

### DISCUSSION THREAD

**Discussion by jaredkrinke** [BattlesnakeOfficial/feedback#237] [Bug Reports]
I'm running "rules" locally, but it looks like the "latency" field isn't set (well, it's always "0") on the "you" Battlesnake object. It is, however, set on the objects in the "snakes" field of "board", so I'm using that as a workaround.

Let me know if you need more info. I searched and couldn't find any discussion of this. I'm on my phone, so I don't have the full JSON handy at the moment.

**Comment by bvanvugt** [comment]
Huh. Feels like a bug. Is anyone else able to confirm this? Maybe @robbles or @coreyja?

**Reply by rob-odwyer** [reply]
I was about to say "We never implemented latency measurement in the local rules CLI", but looking at the code now, it looks like we did, and the object for `you` should have the same info as `board.snakes`.

I'll have to try it out locally to see if I can reproduce. @jaredkrinke is it possible that your local server is just responding in less than a millisecond?

**Reply by jaredkrinke** [reply]
> I'll have to try it out locally to see if I can reproduce. @jaredkrinke is it possible that your local server is just responding in less than a millisecond?

I'm seeing non-zero (and reasonable looking) latency values in my snake's object under "board". I didn't specifically measure latency on my server, but I'm pretty sure it's well over 1ms.
